lexibank_lieberherrkhobwa.py

```diff
@@ -91,7 +88,6 @@ def cmd_install(self, **kw):
                                 Parameter_ID=concept_by_index[cid],
                                 Form=form,
                                 Value=ipa,
```

Comment:
There's still a problem: the value should be the word you find in the data, the form is the processed part (i.e., the IPA), and the segments are generated.

Reply:
so you should probably swap it and make sure the value is really the thing we find in the source.

Reply:
Like this? Value/original data make sense to me.